application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json

```diff
@@ -17,6 +17,7 @@
       }
     },
     "bundleGroupInfo": {
+      "contactUsInfo": "Si prega di contattare l'autore per ulteriori informazioni",
```

Comment:
@GermanoGiudici no need to review this entire PR but if you can review these translations I could use an Italian eye on them. I can show you the context when we talk.